Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
@DomT4 I genuinely appreciate the work you and @chrmoritz have done and continue to do. That said, this PR requires a maintainer to merge it and I don't think it's unreasonable to have at least me or @ilovezfs at the very least understand this change rather than merging it blindly. I'll defer to @ilovezfs on this point.